README.md

```diff
@@ -1 +1,194 @@
-# backend_capstone
\ No newline at end of file
+<div align="center">
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍